stable-baselines3/callbacks.py

```diff
@@ -0,0 +1,52 @@
+from stable_baselines3.common.callbacks import BaseCallback
+from stable_baselines3.common.results_plotter import load_results, ts2xy
+
+import os
+import numpy as np
+
+
+class SaveOnBestTrainingRewardCallback(BaseCallback):
```

Comment:
Optional: Keep a single file implementation and move class to sb3.py? Not sure what protocol is since linear scheduler function is in sb3.py. Move both to their own file or keep everything in one file.

Reply:
I see your point. For simplicity and because I'm almost sure we will only have one callback. I am going ahead and moving it to the sb3.py